graspologic/utils/utils.py

```diff
@@ -974,7 +977,7 @@ def remap_node_ids(
     node_id_dict = dict()
     graph_remapped = type(graph)()
 
-    for source, target, weight in graph.edges(data=weight_attribute):
+    for source, target, weight in graph.edges(data=weight_attribute, default=weight_default):
```

Comment:
Use specified weight if a weight attribute is not found for a given edge.

Reply:
do an nx.is_weighted() on it first and warn on no weight - if they expected their graph to be weighted and mistyped the weight attribute on add or something, they won't even know we're ignoring a weight